hack/alpha-build-machinery/doc.go

```diff
@@ -1,14 +1,3 @@
 // required for gomod to pull in packages.
 
 package alpha_build_machinery
-
-// this is a dependency magnet to make it easier to pull in the build-machinery.  We want a single import to pull all of it in.
-import (
-	_ "github.com/openshift/library-go/alpha-build-machinery/make"
```

Comment:
why?

Reply:
because api doesn't depend on library-go, or at least I can't find anything at openshift/api that depends on library-go

Reply:
this will break the Makefile

Reply:
Can you elaborate? I tried update/verify targets, they continue to work.

Reply:
actually we have copy pasted it for openshift/api to avoid the dependency so this might be the only repo that's ok

Reply:
https://github.com/openshift/api/blob/master/Makefile#L5

Reply:
I still don't understand.

`./hack/alpha-build-machinery/make/` != `/library-go/alpha-build-machinery/make`

Reply:
It is a snapshot to avoid having openshift/api depend on openshift/library-go, other repositories vendor it and reference the vendored path. The build machinery might get its own repo one day to properly fix it.

Reply:
> to avoid having openshift/api depend on openshift/library-go

That's exactly why I removed these lines.

Reply:
yeah, I didn't realize we made this repo special when I originally commented